Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.9-slim-buster as base
+
+WORKDIR /DevOps-Course-Starter
+COPY poetry.lock pyproject.toml /DevOps-Course-Starter/
+
+RUN apt-get update && pip install --upgrade pip && pip install poetry
+
+RUN poetry config virtualenvs.create false
+
+COPY . /DevOps-Course-Starter/
```

Comment:
Copying the folder is fine, but should _definitely_ be combined with having a `.dockerignore` file to prevent secrets etc. from being copied into the image